src/app.ts

```diff
@@ -36,6 +37,7 @@ passport.use(jwt)
 
 app.use(responseHandler)
 //**All routers here*/
+app.use('/', loginRouter)
```

Comment:
I think you should name this route `/login/local` here instead inside the router file. So inside `loginRouter` you can:
`router.post('/', controller)`